packages/stream_chat_flutter/lib/src/message_list_view.dart

```diff
@@ -1008,19 +1008,20 @@ class _MessageListViewState extends State<MessageListView> {
             break;
           case ReturnActionType.reply:
             FocusScope.of(context).unfocus();
-            widget.onMessageSwiped(message);
+            widget.onMessageSwiped!(message);
```

Comment:
We should probably have a null check for widget.onMessageSwiped instead of a bang here